### PR TITLE
[deploy] make deploy --target case sensitive

### PR DESCRIPTION
**Pull request by Zertsov**

### CODE REVIEW

File: packages/cli/src/util/parse-target.ts [hunk 1]
```diff
@@ -23,11 +23,9 @@ export default function parseTarget<FlagName extends string>({
   }
 
   if (typeof targetFlagValue === 'string') {
-    const lowerCaseTarget = targetFlagValue.toLowerCase();
-    output.debug(`Setting target to ${lowerCaseTarget}`);
-    return lowerCaseTarget;
+    return targetFlagValue
```
Comment (TooTallNate) [inline packages/cli/src/util/parse-target.ts]: Needs a changeset. Also, I'm curious why the pre-commit formatter didn't run? (missing semicolon)

**Review by TooTallNate — CHANGES_REQUESTED** [review]

File: packages/cli/src/util/parse-target.ts (hunk 1 above)
Comment (Zertsov) [reply packages/cli/src/util/parse-target.ts]: yeah I just found that too 🤔 

I'll get the changeset made in the next commit because I forgot about that 🙃

**Review by trek — DISMISSED** [review]

File: packages/cli/src/util/parse-target.ts [hunk 2]
```diff
@@ -23,9 +23,7 @@ export default function parseTarget<FlagName extends string>({
   }
 
   if (typeof targetFlagValue === 'string') {
-    const lowerCaseTarget = targetFlagValue.toLowerCase();
```
Comment (trek) [inline packages/cli/src/util/parse-target.ts:26]: Why'd we remove the log?

Comment (Zertsov) [reply packages/cli/src/util/parse-target.ts:26]: Honestly I'm still recovering from sickness so I don't have a great reason. I can add it back

**Review by TooTallNate — APPROVED** [review]

**Pull request merged**